services/zendesk.rb

```diff
@@ -1,18 +1,20 @@
-require 'zendesk2'
+require 'zendesk_api'
 require 'date'
 
 class Zendesk
   def initialize
-    options = {
-      :username => SupportApp.zendesk_username,
-      :token => SupportApp.zendesk_token,
-      :url => SupportApp.zendesk_url
-    }
-    @client = Zendesk2::Client.new(options)
+    @client = ZendeskAPI::Client.new do |config|
+      config.username = SupportApp.zendesk_username
+      config.token = SupportApp.zendesk_token
+      config.url = SupportApp.zendesk_url
+    end
   end
 
   def incidents_for_the_past_week
-    incidents_group = 'Incident response and tuning'
-    @client.tickets.search(query: "created>7days group:\"#{incidents_group}\"").count
+    @client.search(
```

Comment:
This should just be a constant. Also, if we use single quotes we won't have to escape the `"`.